tests/test_sample_size/test_variables.py

```diff
@@ -0,0 +1,121 @@
+import unittest
```

Comment:
naming nit: rename this file to test_metrics.py so it is consistent with the actual file